relocator/chart.go

```diff
@@ -78,6 +79,10 @@ func RelocateChartDir(chartPath string, prefix string, opts ...RelocateOption) e
 	if err != nil {
 		return err
 	}
+	err = relocateCarvelBundle(chartPath, prefix)
```

Comment:
I would put the check for if the file exist here:

```
         carvelLockFile := chartRoot + "/.imgpkg/images.yml"
	 if !utils.FileExists(filepath.Join(chartRoot, ".imgpkg/images.yml")) {
                err = relocateCarvelBundle(chartPath, prefix)
	
```

Also, you could extend chartutils.Chart to have a IsCarvelBundle() method that does the checking and do:

```
 if chart.IsCarvelBundle() {
                err = relocateCarvelBundle(chartPath, prefix)
```

Also, you can pass chart instead of chartPath and get the chart path as chart.RootDir(), but that is just a tiny detail, is just fine using chartPath

Reply:
I did apply some of these feedback. Didn't do the `hasCarvelBundle` thing, but it might be a good addition.